components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -0,0 +1,319 @@
+import argparse
+import celery
+import datetime
+import logging
+import os
+import sys
+import time
+from contextlib import closing
+from pathlib import Path
+
+import msgpack
+import zstandard
+from pydantic import ValidationError
+
+from clp_package_utils.general import CONTAINER_INPUT_LOGS_ROOT_DIR
+from clp_py_utils.clp_config import (
+    CLPConfig,
+    COMPRESSION_JOBS_TABLE_NAME,
+    COMPRESSION_TASKS_TABLE_NAME
+)
+from clp_py_utils.clp_logging import get_logger, get_logging_formatter, set_logging_level
+from clp_py_utils.compression import validate_path_and_get_info
+from clp_py_utils.core import read_yaml_config_file
+from clp_py_utils.sql_adapter import SQL_Adapter
+from job_orchestration.executor.compress.fs_compression_task import compress
+from job_orchestration.scheduler.constants import (
+    CompressionJobStatus,
+    CompressionTaskStatus
+)
+from job_orchestration.scheduler.compress.partition import PathsToCompressBuffer
+from job_orchestration.scheduler.job_config import ClpIoConfig
+from job_orchestration.scheduler.scheduler_data import (
+    CompressionJob,
+    CompressionTaskSuccessResult,
+    CompressionTaskFailureResult,
+)
+
+# Setup logging
+logger = get_logger(__name__)
```

Comment:
```suggestion
logger = get_logger("compression-scheduler")
```

Sorry, I forgot using `__name__` when called as a script will just print `__main__`.